lib/puppet/pops/lookup/module_data_provider.rb

```diff
@@ -66,7 +66,7 @@ def assert_config_version(config)
       if Puppet[:strict] == :error
         config.fail(Issues::HIERA_VERSION_3_NOT_GLOBAL, :where => 'module')
       else
-        Puppet.warn_once(:hiera_v3_at_module_root, config.config_path, _('hiera.yaml version 3 found at module root was ignored'))
+        Puppet.warn_once(:hiera_v3_at_module_root, config.config_path, 'hiera.yaml version 3 found at module root was ignored', config.config_path)
```

Comment:
Looks like we lost the externalization wrapper - would you mind adding back and also in environment_data_provider as it looks like it was missed the first time around?